src/test/resources/Features/OCD-2834-SimplifiedEditingOfListingBasicData.feature

```diff
@@ -29,12 +29,104 @@ Scenario: User logs in as ROLE and edits a listing
-    Then I see "Improper format" error message for Mandatory Disclosures URL filed 
+    Then I see "Improper format" error message for Mandatory Disclosures URL filed
 
 Scenario: User logs in as ROLE and edits a listing
     Given I'm logged in as "ROLE_ONC"
     When I am on listing details page of listing with ID "3384"
     And I click on edit certified product link
-    And I set the Report File Location filed
+    And I set the Report File Location field
     Then I see "not a valid URL." error message for Report File Location
 
+Scenario: Verify error message is displayed when the Certification Date field is left blank
+    Given I'm logged in as "ROLE_ONC"
+    When I am on listing details page of listing with ID "3384"
+    And I click on edit certified product link
+    And I set the certification date field to ""
+    Then I see error message "Field is required" for Certification date
+
+Scenario: Verify error message is displayed when the Product Code field is left blank in the CHPL Product Number
+    Given I'm logged in as "ROLE_ONC"
+    When I am on listing details page of listing with ID "9268"
+    And I click on edit certified product link
+    And I set the product code field to ""
+    Then I see error message "Product Code is required" for CHPL Product Number
+
+Scenario: Verify error message is displayed when the Version Code field is left blank in the CHPL Product Number
+    Given I'm logged in as "ROLE_ONC"
+    When I am on listing details page of listing with ID "9268"
+    And I click on edit certified product link
+    And I set the version code field to ""
+    Then I see error message "Version Code is required" for CHPL Product Number
+
+Scenario: Verify error message is displayed when the ICS field is left blank in the CHPL Product Number
+    Given I'm logged in as "ROLE_ONC"
+    When I am on listing details page of listing with ID "9268"
+    And I click on edit certified product link
+    And I set the ICS field to ""
+    Then I see error message "ICS Code is required" for CHPL Product Number
+
+Scenario Outline: Verify error message is displayed when one/two/three characters/numbers entered in the Product Code field for the CHPL Product Number
+    Given I'm logged in as "<ROLE>"
+    When I am on listing details page of listing with ID "<listing id>"
+    And I click on edit certified product link
+    And I set the product code field to "<value>"
+    Then I see error message "<Error Message>" for CHPL Product Number
+    Examples:
+      |ROLE      |listing id|value|Error Message                                                                     |
+      |ROLE_ADMIN|9268      |a    |Product Code must consist of letters, numbers and/or "_", and be 4 characters long|
+      |ROLE_ONC  |9268      |ab   |Product Code must consist of letters, numbers and/or "_", and be 4 characters long|
+      |ROLE_ACB  |9268      |abc  |Product Code must consist of letters, numbers and/or "_", and be 4 characters long|
```

Comment:
Good examples, but should also check with more than 4 characters, and one with four characters but that has bad values, like `AB#$` or something

Should also do something similar with Version and Product codes